File: MODUL 4/C/PRAK404-2310817320006-Alya_Rosaan.c
#include<stdio.h>

int main(){
    int x;
    float y, z;
    while (x != 5) {
    printf("pilih program : \n");
    printf("1. penjumlahan \n");
    printf("2. pengurangan \n");
    printf("3. perkalian \n");
    printf("4. pembagian \n");
    printf("5. exit \n");
    printf("masukkan pilihan : ");
    scanf("%d", &x);

    if (x == 5){
        printf("terimakasih telah menggunakan kalkulator alya");
        break;
    } else if (x > 5 || x <= 0){
        printf("input anda salah, silahkan coba lagi\n\n");
        continue;
    } else {
        printf("masukkan nilai pertama : "); 
        scanf("%f", &y);
        printf("masukkan nilai kedua : ");
        scanf("%f", &z);
        if (x == 1) {
            float hasil_1 = y + z;
            printf("hasil penjumlahan antara: %.2f dengan %.2f adalah %.2f\n\n", y, z, hasil_1);
        }else if (x == 2){
            float hasil_2 = y - z;
            printf("hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n\n", y, z, hasil_2);
        } else if (x == 3){
            float hasil_3 = y * z;
            printf("hasil perkalian antara %.2f dengan %.2f adalah %.2f\n\n", y, z, hasil_3);
        } else if (x == 4){
            float hasil_4 = y / z;
            printf("hasil pembagian antara %.2f dengan %.2f adalah %.2f\n\n", y, z, hasil_4);
        } 
    }
    }
}

File: MODUL 4/PYTHON/PRAK404-2310817320006-Alya_Rosaan.py
while True:
    print("pilih program")
    print("1. penjumlahan")
    print("2. pengurangan")
    print("3. perkalian")
    print("4. pembagian")
    print("5. exit")
    print("masukkan pilihan : ", end=" ")
    x = int(input())
    if x == 5:
        break
    elif x > 5:
        print("input anda salah, silahkan coba lagi")
        continue
    print("masukkan nilai pertama :", end=" ") 
    y = float(input())
    print("masukkan nilai kedua :", end=" ")
    z = float(input())
    if x == 1:
        hasil_1 = y + z
        print(f"hasil penjumlahan antara {y:.2f} dengan {z:.2f} adalah", hasil_1)
    elif x == 2:
        hasil_2 = y - z
        print(f"hasil pengurangan antara {y:.2f} dengan {z:.2f} adalah", hasil_2)
    elif x == 3:
        hasil_3 = y * z
        print(f"hasil perkalian antara {y:.2f} dengan {z:.2f} adalah", hasil_3)
    elif x == 4:
        hasil_4 = y / z
        print(f"hasil pembagian antara {y:.2f} dengan {z:.2f} adalah", hasil_4)
print("terimakasih telah menggunakan kalkulator alya")        
